File: cs/Iris/ExpressionElement/Instruction/SetVariableInstruction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Iris.SimpleExpressionElement;

namespace Iris.ExpressionElement.Instruction
{
    public class SetVariableInstruction : InstructionExpression
    {
        public CharacterVariableExpression variable;
        public SimpleExpression value;

        public SetVariableInstruction()
        {

        }
    }
}

File: java/Iris/src/com/rokuan/iris/instruction/SetVariableInstruction.java
package com.rokuan.iris.instruction;

import com.rokuan.iris.expression.CharacterVariableExpression;
import com.rokuan.iris.expression.SimpleExpression;

public class SetVariableInstruction extends InstructionExpression
{
	public CharacterVariableExpression variable;
	public SimpleExpression value;

	public SetVariableInstruction()
	{

	}
}
